<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="PointRendererWidget" name="pointRendererWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_phong">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Phong Shader</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">group_effects</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_reload_shaders">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>reload shaders</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_jfpbr">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>JF-PBR</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">group_effects</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="check_trackball">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw trackball</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_first_max_distance">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.080000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_first_max_distance">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>max distance</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PointRendererWidget</class>
   <extends>QWidget</extends>
   <header>PointRendererWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="group_effects"/>
 </buttongroups>
</ui>
